playbooks/common/openshift-cluster/upgrades/upgrade_nodes.yml

```diff
@@ -29,6 +29,10 @@
       {{ hostvars[groups.oo_first_master.0].openshift.common.admin_binary }} drain {{ openshift.node.nodename | lower }} --force --delete-local-data
     delegate_to: "{{ groups.oo_first_master.0 }}"
 
+  - name: Pause for pod rescheduling during node ugprade
+    pause:
+      seconds: "{{ openshift_node_upgrade_pause | default(30) | int }}"
+
```

Comment:
I can image the following scenario: Cluster has `n` nodes `node1`, `node2`, .... Then:
* The first `node1` is upgraded -> pod `important_pod` from `node1` is "rescheduled" to `node2`.
* The second `node2` is upgraded -> pod `important_pod` from `node2` is "rescheduled" to `node3`.
* The third `node3` is upgraded -> pod `important_pod` from `node3` is "rescheduled" to `node4`.
* and so on

Of course, the `important_pod` can be replaced with a group of important pods. So I would rather wait the timeout after the node is schedulable again. So you give a better chance to pending pods to land back on the node. In case the cluster is highly utilized and there is not enough nodes with sufficient resources to receive all drained pods.